app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -552,6 +552,13 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false,
     )
 
+    var lastInteractionWithReEngageCookieBannerDialogInMS by longPreference(
```

Comment:
naming convention should have MS as Ms as below:
```suggestion
    var lastInteractionWithReEngageCookieBannerDialogInMs by longPreference(
```

Reply:
👍🏽